crypto/rsa/rsa_lib.c

```diff
@@ -1345,7 +1345,9 @@ int EVP_PKEY_CTX_set_rsa_keygen_bits(EVP_PKEY_CTX *ctx, int bits)
     return 1;
 }
 
-int EVP_PKEY_CTX_set_rsa_keygen_pubexp(EVP_PKEY_CTX *ctx, BIGNUM *pubexp)
+static int evp_pkey_ctx_set_rsa_keygen_pubexp_intern(EVP_PKEY_CTX *ctx,
+                                                     BIGNUM *pubexp,
+                                                     int stash)
```

Comment:
Maybe it is just me - but I find the variable name 'stash' to be a bit confusing..

Reply:
I agree, not a great name.

Reply:
Yeah, I didn't use 'copy' (which otherwise seems to me an obvious name choice) because in the provider case there is always a copy made eventually; it's just a question of if we retain the input pointer in the EVP_PKEY_CTX itself.

Would 'no_custody' be better (or just 'custody', although then I'd need to flip the values for clarity)?

Reply:
`copy` seems like a good name.  Or `make_copy` even.